### PR TITLE
Remove multi-line statements for AML workspace

### PR DESCRIPTION
**Pull request by guanyu-240**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Remove multi-line statements because it's not supported in portal UX template editor
*
*

### CODE REVIEW

**Comment by guanyu-240** [comment]
@bmoore-msft Hi Brian, could you take a look? The validation seems to be failed with a retriable error

**Comment by guanyu-240** [comment]
@bmoore-msft Please take a look. Thank you!

File: 201-machine-learning-advanced/azuredeploy.json [hunk 1]
```diff
@@ -401,8 +401,7 @@
               "type": "Microsoft.Network/virtualNetworks/subnets",
               "apiVersion": "2019-09-01",
               "name": "[concat(parameters('vnetName'), '/', parameters('subnetName'))]",
-              "properties": "[if(and(equals(parameters('subnetOption'), 'existing'), not(equals(parameters('privateEndpointType'), 'none'))),
-                              union(reference(variables('subnet'), '2019-09-01'), variables('subnetPolicyForPE')), json('null'))]"
+              "properties": "[if(and(equals(parameters('subnetOption'), 'existing'), not(equals(parameters('privateEndpointType'), 'none'))), union(reference(variables('subnet'), '2019-09-01'), variables('subnetPolicyForPE')), json('null'))]"
```
Comment (bmoore-msft) [inline 201-machine-learning-advanced/azuredeploy.json:404]: this seems to make the code less readable - why the change?

Comment (guanyu-240) [reply 201-machine-learning-advanced/azuredeploy.json:404]: @bmoore-msft When user copy and past the whole file into "ARM Template" in Azure portal, it throws validation error.
